challenge-3/src/findings.ts

```diff
@@ -0,0 +1,34 @@
+import { Finding, FindingSeverity, FindingType } from "forta-agent";
+import { CHAIN_IDS } from "./constants";
+
+export const createEscrowFinding = (balanceArbitrum: string, balanceOptimism: string): Finding => {
+  return Finding.fromObject({
+    name: "MakerDAO's L1 escrow balance of Arbitrum & Optimism",
+    description: `Arbitrum's escrow balance ${balanceArbitrum} - Optimism's escrow balance ${balanceOptimism}`,
+    alertId: "L1-Escrow-Balance",
+    severity: FindingSeverity.Info,
+    type: FindingType.Info,
+    protocol: "Ethereum",
+    metadata: {
+      balanceArbitrum: balanceArbitrum,
+      balanceOptimism: balanceOptimism,
+    },
+  });
+};
+
+export const createInvariantFinding = (escrowBalance: string, l2DaiSupply: string, network: number): Finding => {
+  const networkName = network == CHAIN_IDS.Arbitrum ? "Arbitrum" : "Optimism";
+  return Finding.fromObject({
+    name: "MakerDOA Bridge Invariant Violated",
+    description: `L2Dai.totalSupply() : ${l2DaiSupply} > L1Dai.balanceOf(${networkName}Escrow): ${escrowBalance}`,
+    alertId: "MakerDAO-Invariant-Violated",
```

Comment:
Very small change but I believe alertIds should be all capitalized, same for line 24